pkg/controller/logcollector/logcollector_controller.go

```diff
@@ -221,7 +222,9 @@ func fillDefaults(instance *operatorv1.LogCollector) []string {
 					v1.SyslogLogDNS,
 					v1.SyslogLogFlows,
 				}
-
+				if syslog.LogTypes != nil || len(syslog.Encryption) == 0 {
```

Comment:
Shouldnt this be after of the if-statement of line 216?

And then simply set EncryptionNone if len(syslog.Encryption) == 0 ?

Reply:
Yes.. updated the code